db/models.py

```diff
@@ -1 +1,29 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True, blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.SET_NULL,
+                             null=True, related_name="skill_set")
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True, blank=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.SET_NULL,
+                             related_name="players", null=True)
+    guild = models.ForeignKey(Guild, on_delete=models.SET_NULL,
+                              related_name="players", null=True)
+    created_at = models.DateTimeField(auto_now=True)
```

Comment:
auto_now_add would be more appropriate here - it will populate created_at field on record creation as opposed to auto_now that will update field on every write operation